amundsen_application/base/examples/example_superset_preview_client.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import requests
+import uuid
+
+from requests import Response
+from typing import Any, Dict  # noqa: F401
+
+from amundsen_application.base.base_superset_preview_client import BaseSupersetPreviewClient
+
+DEFAULT_DATABASE_MAP = {
+    'main': 1,
+}
+DEFAULT_URL = 'http://localhost:8088/superset/sql_json/'
+
+
+class SupersetPreviewClient(BaseSupersetPreviewClient):
+    def __init__(self,
+                 *,
+                 database_map: Dict[str, int] = DEFAULT_DATABASE_MAP,
+                 url: str = DEFAULT_URL) -> None:
+        self.database_map = database_map
+        self.headers = {}
+        self.url = url
+
+    def post_to_sql_json(self, *, params: Dict, headers: Dict) -> Response:
+        """
+        Returns the post response from Superset's `sql_json` endpoint
+        """
+        # Create the appropriate request data
+        try:
+            request_data = {}  # type: Dict[str, Any]
+
+            # Superset's sql_json endpoint requires a unique client_id
+            request_data['client_id'] = uuid.uuid4()
+
+            # Superset's sql_json endpoint requires the id of the database that it will execute the query on
+            database_name = 'main'  # OR params.get('database') in a real use case
+            request_data['database_id'] = self.database_map.get(database_name, '')
+
+            # Generate the sql query for the desired data preview content
+            try:
+                schema = 'main'  # OR params.get('schema') in a real use case
+                table_name = 'ab_role'  # OR params.get('tableName') in a real use case
```

Comment:
should we mention that `ab_role` is an existing superset table which serves for demo purpose?

Reply:
Yeah, I agree we should call this our directly in the file in case people don't read the doc.